Sage/carbon
# fix(select): fix column alignment in multi column mode - FE-5238

Fixes #5263 

### Checklist




#### QA



### Testing instructions

Use mac with connected mouse or a windows machine and check this story locally - https://carbon.sage.com/?path=/docs/select--with-multiple-columns
